### PR TITLE
Add translation domain to tracy panel

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

**Comment by enumag** [comment]
@fprochazka Travis failed but the error doesn't seem related. And I didn't get the same error on localhost. Some tests did fail on localhost though for yet another reason, probably some CRLF issues.

File: src/Kdyby/Translation/Diagnostics/Panel.php [hunk 1]
```diff
@@ -235,9 +240,12 @@ public function markUntranslated($id)
 	/**
 	 * @param \Exception|\Throwable $e
 	 */
-	public function choiceError($e)
+	public function choiceError($e, $domain)
 	{
-		$this->untranslated[] = $e;
+		$this->untranslated[] = array(
+			'message' => $message,
```
Comment (fprochazka) [inline src/Kdyby/Translation/Diagnostics/Panel.php]: ehm, typo?

**Comment by fprochazka** [comment]
@enumag I guess new feature of nette/code-checker :-/

**Comment by fprochazka** [comment]
@enumag fixed, could you please rebase this onto master?

**Comment by enumag** [comment]
@fprochazka Thanks! Done with the rebase.

**Comment by fprochazka** [comment]
Looks good, thanks!

**Comment by enumag** [comment]
You're welcome. :-)

**Pull request merged**